emperor/qiime_backports/filter.py

```diff
@@ -115,6 +115,6 @@ def filter_mapping_file(map_data, map_header, good_sample_ids,
     headers.append(map_header[-1])
     result.append(to_keep[-1])
     
-    result = map(list,zip(*result))
+    result = list(map(list,list(zip(*result))))
```

Comment:
actually, if the inner lists don't need to be editable, then you can further simplify to
```python
list(zip(*x))
```